string name
string discrete_value
float64 continuous_value
